stock_release_channel/models/stock_picking.py

```diff
@@ -106,18 +106,43 @@ def _find_release_channel_possible_candidate(self):
             .sorted(key=lambda r: (not bool(r.partner_ids), r.sequence))
         )
 
-    def _get_release_channel_possible_candidate_domain(self):
-        self.ensure_one()
+    def _get_release_channel_possible_candidate_domain_channel(self):
         return [
             ("is_manual_assignment", "=", False),
             ("state", "!=", "asleep"),
+        ]
+
+    def _get_release_channel_possible_candidate_domain_picking(self):
+        return [
             "|",
             ("picking_type_ids", "=", False),
             ("picking_type_ids", "in", self.picking_type_id.ids),
             "|",
             ("warehouse_id", "=", False),
             ("warehouse_id", "=", self.picking_type_id.warehouse_id.id),
+        ]
+
+    def _get_release_channel_possible_candidate_domain_partners(self):
```

Comment:
as you used singular or channel and picking domains
```suggestion
    def _get_release_channel_possible_candidate_domain_partner(self):
```